cs251tk/student/markdownify/process_file.py

```diff
@@ -28,11 +28,19 @@ def get_file(filename, results, options):
     return True
 
 
-def compile_file(filename, steps, results, supporting_dir):
+def compile_file(filename, steps, results, supporting_dir, basedir, web, student):
+    server_path = ''.join([
```

Comment:
```suggestion
    server_path = ' '.join([
```

Reply:
By joining on `" "`, we don't need to have the trailing spaces on the following lines.